errors.go

```diff
@@ -30,9 +30,3 @@ const ErrRejected = errorType("response rejected")
 
 // ErrUnsupported indicates an operation is not supported by the transport protocol
 const ErrUnsupported = errorType("unsupported")
-
-// ErrDisconnected indicates the other peer may have hung up and you should try restarting the channel.
-const ErrDisconnected = errorType("other peer appears to have hung up. restart Channel")
```

Comment:
ErrDisconnected and ErrRemoved are no longer used (with this PR we try to reconnect automatically)